pvr.nextpvr/changelog.txt

```diff
@@ -1,3 +1,9 @@
+v8.2.4
+- Update timer from a recurring recording
+- Remove extrparameter on log call
```

Comment:
`extrparameter`? Should this read `extra parameter`?

Reply:
Absolutely.  Should be ok now.